src/reference-manual/statements.Rmd

```diff
@@ -895,27 +902,30 @@ The order in which elements of `ys` are visited is defined for container types a
 
 * `matrix`: elements visited in column-major order, `y` is of type `double`
 
-* `T[]`: elements visited in order, `y` is of type `T`.
+* `array[] T`: elements visited in order, `y` is of type `T`.
 
-Consequently, if `ys` is a two dimensional array `real[ , ]`, `y` will
-be a one-dimensional array of real values (type `real[]`).  If 'ys` is
+Consequently, if `ys` is a two dimensional array `array[,] real`, `y` will
+be a one-dimensional array of real values (type `array[] real`).  If 'ys` is
 a matrix, then `y` will be a real value (type `real`).  To loop over
 all values of a two-dimensional array using foreach statements would
 require a doubly-nested loop,
 
-```
-real yss[2, 3];
-for (ys in yss)
-  for (y in ys)
-    ... do something with y ...
+```stan
+array[2, 3] real yss;
+for (ys in yss) {
+  for (y in ys) {
+    // ... do something with y ...
+  }
+}
 ```
 
 whereas a matrix can be looped over in one foreach statement
 
-```
+```stan
 matrix[2, 3] yss;
-for (y in yss)
-   ... do something with y...
+for (y in yss) {
+   // ... do something with y...
```

Comment:
Is there a reason you added a comment here?  The doing something with y shouldn't be commented out in the actual code as it needs to do something with y!

Reply:
We had about half and half of ellipses and “so something”s being commented out and not, I went in favor so that the examples are reasonably close to something that would parse straight off, but I’m not attached to either choice. I can make it consistent either way

Reply:
OK.  It'd be great if it were consistent.  I see the argument for having something that parses with the formatter.   Do you want to do that as part of this PR?

Reply:
Might as well do it now